docker/build.sh

```diff
@@ -123,26 +128,13 @@ if $PRINT_HELP; then
   exit 0
 fi
 
-# Configure script using arguments.
-if [[ $DOCKER_REGISTRY == "dockerhub" ]]; then
-  DOCKER_PROJECT="${DOCKER_PROJECT:-broadinstitute}"
-  REPO="${DOCKER_PROJECT}"
-  IMAGE="${REPO}/${TARGET}"
-  DOCKER_REMOTES_BINARY="docker"
-  NOTEBOOK_REPO="${NOTEBOOK_REPO:-$REPO}"
-elif [[ $DOCKER_REGISTRY == "gcr" ]]; then
-  DOCKER_PROJECT="${DOCKER_PROJECT:-$(gcloud config get-value project)}"
-  # Domain scoped project IDs need to be modified to work with GCR.
-  REPO="gcr.io/$(sed "s_:_/_" <<< "${DOCKER_PROJECT}")"
-  IMAGE="${REPO}/${TARGET}"
-  DOCKER_REMOTES_BINARY="gcloud docker --"
-  NOTEBOOK_REPO="${NOTEBOOK_REPO:-$REPO}"
-else
-  echo "The docker registry must be either 'dockerhub' or 'gcr'"
-  echo "Provided value: ${DOCKER_REGISTRY} is not allowed."
-  exit 1
-fi
-
+DOCKERHUB_REGISTRY=${DOCKERHUB_REGISTRY:-broadinstitute}
+GCR_REGISTRY=${GCR_REGISTRY:-gcr.io/broad-dsp-gcr-public}
+DOCKER_REMOTES_BINARY="docker"
+GCR_REMOTES_BINARY="gcloud docker --"
+NOTEBOOK_REPO="${NOTEBOOK_REPO:-$REPO}"
```

Comment:
I don't think `$REPO` is a variable anymore with the refactoring. Maybe this line could be changed to:
```
NOTEBOOK_REPO="${NOTEBOOK_REPO:-$GCR_REGISTRY}"
```
Then we wouldn't even have to specify an `-n` parameter in `jenkins_build.sh`.

Reply:
would we still want to have that option though in case we want to push notebooks image to a different registry in the future?